engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -5730,6 +5735,22 @@ std::unique_ptr<expr_t> demon_hunter_t::create_expression( util::string_view nam
       return this->cooldown.eye_beam->create_expression( "remains" );
     }
   }
+  else if ( util::str_compare_ci( name_str, "dh.darkglare_boon_cdr_roll" ) )
+  {
```

Comment:
This could probably just be simplified down to
return make_mem_fn_expr( name_str, *this, &demon_hunter_t::darkglare_boon_cdr_roll );